src/fu-udev-device.c

```diff
@@ -103,32 +103,6 @@ fu_udev_device_get_sysfs_attr_as_uint8 (GUdevDevice *udev_device, const gchar *n
 	return tmp;
 }
 
-static void
-fu_udev_device_dump_internal (GUdevDevice *udev_device)
```

Comment:
This is kinda useful for plugin development. I wonder if we should put this into the default `to_string()` implementation, perhaps covered by a `g_getenv("FU_UDEV_DEVICE_DEBUG")` or something.

Reply:
Yeah I figured it was useful for something like that, but I didn't want to export a symbol to the new library if nothing is using it.